packages/packages/commonwell-cert-runner/src/patient-management.ts

```diff
@@ -44,7 +44,7 @@ export async function patientManagement(
     patient.details.name[0].family[0],
     patient.details.birthDate,
     patient.details.gender.code,
-    patient.details.address[0].zip
+    patient.details.address[0].zip ?? undefined
```

Comment:
why are we changing this? zip might not be required for doc refs, but afaik it is required for patient/persons/searches

Reply:
My thinking was that types/compilation would save us here, and was able to search Patient w/o zip.

But you're right, zip is required to add Person, so I'll need to make this optional only for that usage.